tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -796,6 +796,7 @@ def test_scontrol_reboot(
         slurm_commands,
         ["queue1-dy-cr1-1", "queue1-dy-cr1-2"],
         "idle",
+        wait_fixed_secs=5,
```

Comment:
The wait fixed is the one where it retries after the time we specify. Stop_max_delay is the one which is the overall seconds after which it will stop retrying

Reply:
For `wait_for_compute_nodes_states` the default values of wait_fixed is 20s which is high as we cannot get the node in state of idle and end up getting Mixed state